types/jasmine/jasmine-tests.ts

```diff
@@ -500,6 +500,53 @@ describe("A spy, when configured to fake a series of return values", () => {
     });
 });
 
+describe("A spy, when configured to fake a promised return value", () => {
+    const bar = 10;
+    const foo = {
+        getAsyncBar: () => {
+            return Promise.resolve(bar);
+        }
+    };
+
+    it("verifies return value type", () => {
+        spyOn(foo, "getAsyncBar").and.resolveTo(745);
+        spyOn(foo, "getAsyncBar").and.resolveTo("42"); // Is an error with TS 3.1+ typings.
```

Comment:
Not sure what the meaning of this test case is? It only creates spies but does nothing else

Reply:
Meaning of the test case is to give a compile error if resolveTo() is removed from typings. Remember the tests are for the typings, not for the jasmine library (at least this is my informed guess by looking at existing tests). The exact same set of tests are there for and.returnValue().

Goal here was to be roughly consistent with the tests for and.returnValue, so unless you want to turn this into "refactor tests", I'd leave it as-is.